app/recordtransfer/forms/transfer_forms.py

```diff
@@ -889,23 +889,23 @@ def __init__(self, *args, **kwargs):
         choices = [(None, "Select a group")] + [
             (group.uuid, group.name) for group in user_submission_groups
         ]
-        self.fields["group_id"].choices = choices
-        self.fields["group_id"].initial = None
+        self.fields["group_uuid"].choices = choices
+        self.fields["group_uuid"].initial = None
 
     def clean(self) -> dict:
         """Check that chosen group exists and is owned by the user."""
         cleaned_data = super().clean()
-        group_id = cleaned_data.get("group_id")
+        group_uuid = cleaned_data.get("group_uuid")
 
-        if (
-            group_id
-            and not SubmissionGroup.objects.filter(created_by=self.user, uuid=group_id).exists()
-        ):
-            self.add_error("group_id", "Invalid group selected.")
+        group = SubmissionGroup.objects.filter(created_by=self.user, uuid=group_uuid).first()
+        if not group:
+            raise ValidationError({"group_uuid": "Invalid group selected."})
```

Comment:
Should be:

```suggestion
            self.add_error("group_uuid", "Invalid group selected.")
```

Though if there was a specific reason why you raised an error here you can ignore this suggestion.

Reply:
I raised a ValidationError on purpose there since I didn't want to include the `submission_group` in the `cleaned_data`. Showing an error is no longer necessary as per https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/387/commits/d26c6205bab660668aaa39355378288d482abfdd